discordrb.json

```diff
@@ -57,6 +57,18 @@
             "voiceReceive": "no",
             "voiceSend": "yes",
             "voiceStateUpdate": "yes"
+        },
+        "misc": {
+            "supportsUserLogin": "yes",
+            "supportsBotLogin": "yes",
```

Comment:
Whether the library supports logging in as a bot account. (If you think this is useless, there have been libs specifically made for custom clients in the past - see DiscordUWP8 - and there may be ones in the future). Should be `yes` or `no`.